README.md

```diff
@@ -286,7 +286,18 @@ Available launch-file options:
 
 ### Example 2: "Industrial Robots with only one interface" (Gazebo simulation)
 
-- **TBA**
+- Launch file: ros2_control_demo_bringup rrbot_system_position_only_gazebo.launch.py
+ 
+- Command interfaces:
+  - joint1/position
+  - joint2/position
+- State interfaces:
+  - joint1/position
+  - joint2/position
+
+Available controllers:
+  - `joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active`
+  - `joint_trajectory_controller[joint_trajectory_controller/JointTrajectoryController] active`
 
```

Comment:
Do se need here some additional steps how to run example or is it self-explanatory? How hard was for you to understand everything?

Reply:
I add the necessary steps to get it started in the readme and no more steps are needed except for the changes in this pull request.

Regarding your question about difficulty in understanding:

I think that since multiple people contribute so differences in naming convention, file structure/layouts results as a consequence. Like for example

1) Launch File Layout: What I found confusing was different layouts for the files used in the different packages.  I think that once a layout for the launch files can be agreed upon then all the launch files in the example can be refactored to a common layout so that the learner can focus only on the part which changed in different launch files. I made an [example layout](https://github.com/haider8645/ros2_launch_tutorial/blob/master/launch/ros2.launch.py) by combing different layouts  to implement some advanced logic (if needed) in the launch file using the context. This will help use the full potential of the launch file module capabilities. 

2) There were/are some naming discrepancies which I will try to rectify in further pull requests. For example the changes you saw in PR#190

3) I think helper comments can be added to the configs and specially the gazebo files for people who just start in ROS2. I guess that the assumption while creating this package docu was that the person has some ROS1 knowledge about gazebo ros, gazebo_ros_control etc. but since many people start now with ROS2, they might find it hard to follow the complexity. The complexity because you need the name joint names in the gazebo xacro, controller yml, and urdf.. so it can be hard to follow it for new comers. 

PS: I will be working with this package so I will try to create new PR's to enhance the docu and understandability as a user/learner myself.

Reply:
you are very welcome to do that, but please do PR against master.

Reply:
@destogl Helloo, I updated everything for the master and changed the base to master. However, I guess because I was working on the foxy and I merged master into it there are those 3 commits from foxy branch(1 from you, 2 from bot) also now part of this PR. Do you know how I can easily remove them?

Reply:
The only I can think of to remove those commits is to switch to master, update it to latest state and cherry pick my commits one at a time. Is the a possible way forward?

Reply:
Turns out that those commits do not really appear in the File Changed. The file changed now look good for my pov.

Reply:
@destogl could we please merge this? It has been open for a long time now